res/css/photon-components.css

```diff
@@ -121,3 +154,139 @@
 .photon-message-bar-error .photon-button:hover:active {
   background-color: var(--red-90);
 }
+
+/* Label */
+
+.photon-label,
+.photon-label-micro {
+  display: inline-flex;
+  color: var(--grey-90);
+}
+
+.photon-label {
+  padding: 3px 0;
+  font-size: 13px;
+}
+
+.photon-label-micro {
+  padding: 3px 0;
+  font-size: 11px;
+}
+
+/* Checkboxes */
+
+.photon-checkbox,
+.photon-checkbox-micro {
+  position: relative;
+  padding: 0;
+  border: 1px solid var(--grey-90-a30);
+  -webkit-appearance: none;
+  -moz-appearance: none;
+  appearance: none;
+  background-color: var(--grey-90-a10);
+  border-radius: 2px;
+}
+
+.photon-checkbox {
+  width: 16px;
+  height: 16px;
+  margin: 0 4px;
+}
+
+.photon-checkbox-micro {
+  width: 13px;
+  height: 13px;
+  margin: 0 3px;
+}
+
+.photon-checkbox:hover,
+.photon-checkbox-micro:hover {
+  background-color: var(--grey-90-a20);
+}
+
+.photon-checkbox:active,
+.photon-checkbox-micro:active {
+  background-color: var(--grey-90-a30);
+}
+
+.photon-checkbox:checked,
+.photon-checkbox-micro:checked {
+  border: none;
+  background-color: var(--blue-60);
+}
+
+.photon-checkbox:checked {
+  background-image: url(../img/svg/checkbox-check-16.svg);
+}
+
+.photon-checkbox-micro:checked {
+  background-image: url(../img/svg/checkbox-check-13.svg);
```

Comment:
Is it possible to use the sams SVG file for both cases ? I thought that's the point of using a vector image format...
(same remark for the other similar cases)

Reply:
This is not the same graphic. I modified the shape to fit better in the space.